<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMMGeneratorSettingsEditorWidget</class>
 <widget class="QFrame" name="CMMGeneratorSettingsEditorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>261</width>
    <height>195</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CMMGeneratorSettingsEditorWidget</string>
  </property>
  <layout class="QFormLayout" name="mainLayout">
   <property name="horizontalSpacing">
    <number>5</number>
   </property>
   <property name="verticalSpacing">
    <number>5</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="lblCPUMachinePECount">
     <property name="text">
      <string>CPU machine PE count:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QSpinBox" name="sbCPUMachinePECount">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>1048576</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblCPUMachinePERating">
     <property name="text">
      <string>CPU machine PE rating:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="sbCPUMachinePERating">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>1048576</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblGPUMachinePECount">
     <property name="text">
      <string>GPU machine PE count:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSpinBox" name="sbGPUMachinePECount">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>1048576</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lblGPUMachinePERating">
     <property name="text">
      <string>GPU machine PE rating:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QSpinBox" name="sbGPUMachinePERating">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>1048576</number>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lblResourceBaudRate">
     <property name="text">
      <string>Resource baud-rate (MBps):</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QDoubleSpinBox" name="sbResourceBaudRate">
     <property name="decimals">
      <number>7</number>
     </property>
     <property name="minimum">
      <double>1.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1000000000.000000000000000</double>
     </property>
     <property name="value">
      <double>10000.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="lblLinkBaudRate">
     <property name="text">
      <string>Link baud-rate (MBps):</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QDoubleSpinBox" name="sbLinkBaudRate">
     <property name="decimals">
      <number>7</number>
     </property>
     <property name="minimum">
      <double>1.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1000000000.000000000000000</double>
     </property>
     <property name="value">
      <double>10000.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="lblLoadOperationCost">
     <property name="text">
      <string>Load operation cost:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QDoubleSpinBox" name="sbLoadOperationCost">
     <property name="decimals">
      <number>7</number>
     </property>
     <property name="minimum">
      <double>0.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1000000000.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="lblSaveOperationCost">
     <property name="text">
      <string>Save operation cost:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QDoubleSpinBox" name="sbSaveOperationCost">
     <property name="decimals">
      <number>7</number>
     </property>
     <property name="minimum">
      <double>0.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1000000000.000000000000000</double>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
